pages/provinces/[provinceSlug]/[contactSlug].tsx

```diff
@@ -4,6 +4,7 @@ import { BackButton } from "../../../components/layout/back-button";
 import { Page } from "../../../components/layout/page";
 import { PageContent } from "../../../components/layout/page-content";
 import { PageHeader } from "../../../components/layout/page-header";
+import ReportButton from "../../../components/ReportButton";
```

Comment:
```suggestion
import ReportButton from "../../../components/report-button";
```

Please rename the file into `report-button.tsx` to comply with our de facto file naming convention.